query_execution/PolicyEnforcer.hpp

```diff
@@ -60,12 +61,14 @@ class PolicyEnforcer {
                  const std::size_t num_numa_nodes,
                  CatalogDatabaseLite *catalog_database,
                  StorageManager *storage_manager,
-                 tmb::MessageBus *bus)
+                 tmb::MessageBus *bus,
+                 WorkerDirectory *worker_directory)
```

Comment:
This is optional, but I would suggest to move `worker_directory` before `bus` under the idea that puts all third party stuff last.